core/ui/fields/field_angle_dropdown.js

```
@@ -33,7 +47,14 @@ Blockly.FieldAngleDropdown.prototype.showEditor_ = function() {
   var svgContainer = goog.dom.createDom('div');
   container.appendChild(svgContainer);
 
-  var dir = this.sourceBlock_.getTitleValue(this.directionTitleName);
+  // Turn right (clockwise) by default.
+  var dir = 'turnRight'
+  if (this.directionTitleName) {
+    dir = this.sourceBlock_.getTitleValue(this.directionTitleName);
+  } else if (this.direction) {
```

I'd throw an error if both `directionTitleName` and `direction` are specified.